modules/quickstart-sap/pages/configure.adoc

```diff
@@ -0,0 +1,61 @@
+[[quickstart-sap-clients]]
+= Configure Clients
+
+{productname} uses formulas with forms to configure your SAP clients.
+There are two formulas with forms that you need to use: one to configure the HANA database, and one to configure the clients into a cluster.
+
+The formulas are provided by packages that you can download from your package manager.
+You need to install the formulas on the {productname} master client.
+When you have installed the package, you can use the {productname} {webui} to enable and configure the formulas.
+As you go through the formula configuration process, you will be able to provide the details of the clients that contain your SAP cluster, to set them up appropriately.
+
+
+[IMPORTANT]
+====
+The order that you enable and configure the formulas is important.
+You must enable, configure, and apply the HANA formula first.
+Then you can enable, configure, and apply the cluster formula.
+If you perform these steps in the wrong order, your SAP installation will fail.
+====
+
+
+To install the SAP formulas, you need the ``SUSE Package Hub 15 SP1 x86_64`` product from {scc}.
```

Comment:
@Loquacity We released it on package Hub as a beta version. The official packages will be part of the SLES for SAP official repositories, so enabling packagehub will not be needed in a near future. On SLES for SAP 15 SP2 the packages are already available as we have ECOs as WIP to backport this to older SLES for SAP 12 SP3 and onwards.